storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -713,6 +714,38 @@ func (self *ReconcileVolume) finalizeVolume(
 	return delayedRequeue(nil)
 }
 
+func (self *ReconcileVolume) refreshDeviceName(
+	ctx context.Context,
+	volume *storagev1alpha1.Volume,
+	pv *corev1.PersistentVolume,
+) (reconcile.Result, error) {
+	path := pv.Spec.PersistentVolumeSource.Local.Path
```

Comment:
Maybe have an assertion that the given PV is actually of type `Local`, i.e. `Path` is not `""`?

Reply:
It's supposed to be PV we created no?
But will do, it doesn't hurt to be extra-safe on that point.